engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-events.html

```diff
@@ -1,7 +1,7 @@
 <section class="nutupane-sub-section">
   <div bst-container-scroll>
     <div infinite-scroll="eventTable.nextPage()" infinite-scroll-container="'.nutupane-sub-section .container-scroll-wrapper'">
-      <table bst-table="eventTable" class="table table-striped">
+      <table bst-table="eventTable" class="table table-striped" ng-if="eventTable">
```

Comment:
I would think initializing the table to an empty object outside the promise would allow you to avoid the ng-if, but you'd have to test it.